test/components/tableTests.ts

```diff
@@ -471,6 +471,30 @@ describe("Tables", () => {
         table.destroy();
         svg.remove();
       });
+
+      it("rowPadding and columnPadding error on invalid input", () => {
+        const svg = TestMethods.generateSVG();
+        const table = new Plottable.Components.Table([[new Plottable.Component(), new Plottable.Component()],
+                                                      [new Plottable.Component(), new Plottable.Component()]]);
+        assert.throws(() => table.columnPadding(-1), "columnPadding must be a non-negative finite value");
```

Comment:
sorry, should have caught this earlier but the proper way to assert error is
`(<any> assert).throws(() => c.foo(), Error, "Error message", "The reason");`
with a hackhack comment referring #2661

also, can you test for `NaN` and number string?